### DISCUSSION TITLE
F-Droid out of date 4.0.0-alpha7

### DISCUSSION THREAD

**Discussion by warp9pnt9** [godotengine/godot-proposals#7286] [Platforms]
Just a ping.  I saw 4.1 was released a while ago.  I noticed Google Play has 4.1 but F-Droid is stuck on 4.0.0-alpha7.  Any plans to update F-Droid?  Are F-Droid releases automated?  Is F-Droid release not supported or being abandonned?  I prefer F-Droid whenever possible and would like to see it prominently featured as a download option on the website, but if Google Play is the only place getting updates, best make that clear.  Otherwise, if we get direct downloads from the Godot website or from GitHub releases or whereever, having a way to check for updates and remind us would be cool.

**Comment by sonic2kk** [comment]
I could be wrong, but I'm not sure the version on F-Droid is official? The [Godot website downloads page](https://godotengine.org/download/android/) only lists the Play Store and a direct download option. If the F-Droid release is officially from the Godot team, it might be worth mentioning it on the website.

I did a quick search (on PC, not on my phone) and it seems like F-Droid [at least has 4.0.1](https://f-droid.org/packages/org.godotengine.editor.v4/). Not sure though, I'm not sure if any Godot packages on F-Droid are official.

<hr>

Having said that, for direct downloads, a way to check for automatic updates would be cool if such a feature is not already present.

**Reply by Calinou** [reply]
> Not sure though, I'm not sure if any Godot packages on F-Droid are official.

The F-Droid packages are not official. As a result, they're updated on a best-effort basis.

**Comment by Sauermann** [comment]
Please have a look here for F-Droid/Godot details: #6224
